packages/disputer/index.js

```diff
@@ -46,32 +46,10 @@ async function run({
   try {
     const { toBN } = web3.utils;
 
-    // If pollingDelay === 0 then the bot is running in serverless mode and should send a `debug` level log.
-    // Else, if running in loop mode (pollingDelay != 0), then it should send a `info` level log.
-    logger[pollingDelay === 0 ? "debug" : "info"]({
```

Comment:
This is mostly rearranging code. Basically I needed to query the collateralCurrency.decimals() before constructing the pricefeed, so that I could detect decimals for the price feed.